mail_tracking/models/mail_tracking_email.py

```diff
@@ -0,0 +1,165 @@
+    _order = 'time desc'
+    _rec_name = 'name'
+    _description = 'MailTracking email'
+
+    name = fields.Char(string="Subject", readonly=True)
+    timestamp = fields.Float(
+        string='UTC timestamp', readonly=True,
+        digits=dp.get_precision('MailTracking Timestamp'))
+    time = fields.Datetime(string="Time", readonly=True)
+    date = fields.Date(string="Date", readonly=True)
+    mail_message_id = fields.Many2one(
+        string="Message", comodel_name='mail.message', readonly=True)
+    mail_id = fields.Many2one(
+        string="Email", comodel_name='mail.mail', readonly=True)
+    partner_id = fields.Many2one(
+        string="Partner", comodel_name='res.partner', readonly=True)
+    recipient = fields.Char(string='Recipient email', readonly=True)
+    sender = fields.Char(string='Sender email', readonly=True)
+    state = fields.Selection([
+        ('error', 'Error'),
+        ('deferred', 'Deferred'),
+        ('sent', 'Sent'),
+        ('delivered', 'Delivered'),
+        ('opened', 'Opened'),
+        ('rejected', 'Rejected'),
+        ('spam', 'Spam'),
+        ('unsub', 'Unsubscribed'),
+        ('bounced', 'Bounced'),
+        ('soft-bounced', 'Soft bounced'),
+    ], string='State', index=True, readonly=True, default=False,
+        help=" * The 'Error' status indicates that there was an error "
+             "when try to sent the email, for example, 'No valid recipient'\n"
+             " * The 'Sent' status indicates that message was succesfully "
+             "sent via outgoing email server (SMTP).\n"
+             " * The 'Delivered' status indicates that message was "
+             "succesfully delivered to recipient Mail Exchange (MX) server.\n"
+             " * The 'Opened' status indicates that message was opened or "
+             "clicked by recipient.\n"
+             " * The 'Rejected' status indicates that recipient email "
+             "address is blacklisted by outgoing email server (SMTP). "
+             "It is recomended to delete this email address.\n"
+             " * The 'Spam' status indicates that outgoing email "
+             "server (SMTP) consider this message as spam.\n"
+             " * The 'Unsubscribed' status indicates that recipient has "
+             "requested to be unsubscribed from this message.\n"
+             " * The 'Bounced' status indicates that message was bounced "
+             "by recipient Mail Exchange (MX) server.\n"
+             " * The 'Soft bounced' status indicates that message was soft "
+             "bounced by recipient Mail Exchange (MX) server.\n")
+    error_smtp_server = fields.Char(string='Error SMTP server', readonly=True)
+    error_type = fields.Char(string='Error type', readonly=True)
+    error_description = fields.Char(
+        string='Error description', readonly=True)
+    bounce_type = fields.Char(string='Bounce type', readonly=True)
+    bounce_description = fields.Char(
+        string='Bounce description', readonly=True)
```

Comment:
Not writing what you don't need is a better one IMHO, but it's just recommendation anyway.